cavern/src/main/java/org/opencadc/cavern/nodes/NodeUtil.java

```diff
@@ -72,6 +72,7 @@
 import ca.nrc.cadc.date.DateUtil;
 import ca.nrc.cadc.reg.Standards;
 import ca.nrc.cadc.reg.client.LocalAuthority;
+import ca.nrc.cadc.util.PropertiesReader;
```

Comment:
I can't remember if checkstyle is run on this repo but it might complain about these unused imports.

Reply:
'gradle checkstyleMain' doesn't work locally (so I'm doubting it's run in github build.) Still - removing these.

Reply:
done